src/blocks/custom_dbus.rs

```diff
@@ -37,11 +37,24 @@ pub struct CustomDBus {
 pub struct CustomDBusConfig {
     pub name: String,
 
+    /// Text to display on startup until the first update is received on the bus.
+    pub initial_text: String,
+
     /// Timeout for clearing the block output after an update (in seconds)
     #[serde(default, deserialize_with = "deserialize_opt_duration")]
     pub timeout: Option<Duration>,
 }
 
+impl Default for CustomDBusConfig {
+    fn default() -> Self {
+        Self {
+            name: Default::default(),
```

Comment:
Shouldn't `name` be a required parameter? (Like `mac` in `bluetooth`)

Reply:
Indeed